packages/x-date-pickers-pro/src/internal/utils/views.tsx

```diff
@@ -0,0 +1,70 @@
+import * as React from 'react';
+import { DateRangeCalendar, DateRangeCalendarProps } from '../../DateRangeCalendar';
+
+/**
+ * We don't pass all the props down to `DateRangeCalendar`,
+ * because otherwise some unwanted props would be passed to the HTML element.
+ */
+export const renderDateRangeView = <TDate extends unknown>({
```

Comment:
If you have a better solution here, I'm interested
At least it is centralized :grimacing:

Reply:
Not sure their is a better way than listing the props that can be passed. The other options would be to stop them when rendering the HTML element which does not seems better